BitwardenShared/Core/Auth/Models/Response/MasterPasswordUnlockKdfResponseModel.swift

```diff
@@ -0,0 +1,19 @@
+// MARK: - MasterPasswordUnlockKdfResponseModel
+
+/// API response model for a user's master password unlock KDF configuration.
+///
+struct MasterPasswordUnlockKdfResponseModel: Codable, Equatable, Hashable {
+    // MARK: Properties
+
+    /// The type of key derivation function algorithm to use.
+    let kdfType: KdfType
+
+    /// The number of iterations for the key derivation function.
+    let iterations: Int
+
+    /// The memory cost parameter for memory-hard KDF algorithms like Argon2.
+    let memory: Int?
+
+    /// The parallelism parameter for KDF algorithms that support parallel computation.
+    let parallelism: Int?
+}
```

Comment:
🤔 I see this is the same as our current `KdfConfig`, should we somehow reuse/change our current objects? If not, should this be conforming to `KdfConfigProtocol` as with any of the other response models that have KDF configuration to ease integration with SDK side KDF objects?